documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -67,10 +67,10 @@ Cruise Control uses _capacity limits_ to determine if certain resource-based opt
 You specify capacity limits for Kafka broker resources in the `brokerCapacity` property in `Kafka.spec.cruiseControl` .
 Capacity limits can be set for the following broker resources in the described units:
 
-* `disk`            - Disk storage in bytes
-* `cpuUtilization`  - CPU utilization as a percent (0-100)
-* `inboundNetwork`  - Inbound network throughput in bytes per second
-* `outboundNetwork` - Outbound network throughput in bytes per second
+* `disk`            - Disk storage in megabytes (Default: 100000)
+* `cpuUtilization`  - CPU utilization as a percentage (Default: 100)
+* `inboundNetwork`  - Inbound network throughput in kilobytes per second (Default: 10000)
+* `outboundNetwork` - Outbound network throughput in kilobytes per second (Default: 10000)
```

Comment:
When I tried to follow this, I struggled a lot with this and I think this PR fixes only some of that. It still says that `disk` is in megabytes while the example below has `100G` which is confusing. I think you should really list here all the available units. Ideally, the units for the storage should match what Kubernetes use.

Reply:
Good point, the units can be many different sizes and use the K8s convention. I have updated the docs to reflect this.